src/local-page/local-page.jsx

```diff
@@ -1,87 +0,0 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
-import { Button, Icon } from 'semantic-ui-react'
```

Comment:
Might as well remove this package now as well (yay!): only used in `local-page` module